zeebe/engine/src/main/java/io/camunda/zeebe/engine/EngineConfiguration.java

```diff
@@ -135,4 +139,12 @@ public EngineConfiguration setEnableAuthorization(final boolean enableAuthorizat
     this.enableAuthorization = enableAuthorization;
     return this;
   }
+
+  public int getDefaultCallActivityMaxDepth() {
+    return defaultCallActivityMaxDepth;
+  }
+
+  public void setDefaultCallActivityMaxDepth(final int defaultCallActivityMaxDepth) {
+    this.defaultCallActivityMaxDepth = defaultCallActivityMaxDepth;
+  }
```

Comment:
🔧 My feeling is that the wording default does not make sense for this configuration option.

```suggestion
  public int getCallActivityMaxDepth() {
    return callActivityMaxDepth;
  }

  public void setCallActivityMaxDepth(final int callActivityMaxDepth) {
    this.callActivityMaxDepth = callActivityMaxDepth;
  }
```